### PR TITLE
FIP-56 Fei v2 Deployment

### PR DESCRIPTION
**Pull request by Joeysantoro**
FIP-56 Fei v2 Deployment

DEPLOY ACTIONS:
1. Deploy TribeMinter
2. Deploy TribeReserveStabilizer
3. Deploy DAI PriceBoundPegStabilityModule
4. Deploy DAI PCV Dripper

DAO ACTIONS:
1. Create TRIBE_MINTER_ADMIN Role
2. Grant TribeReserveStabilizer Admin Role
3. Set TribeMinter Contract Admin Role
4. Grant TRIBE_MINTER the Tribe Minter role
5. Grant Minter Role to DAI PSM
6. Grant PCV Controller to DAI PCVDripController
7. Create PSM_ADMIN_ROLE
8. Withdraw 20m DAI to PSM from Compound
9. Revoke Minter from DPI bonding curve
10. Revoke Minter from RAI bonding curve
11. Remove RAI & DPI bonding curve from Collateralization Oracle
12. Add DAI PSM to Collateralization Oracle
13. Raise ETH BondingCurve buffer to 75bps
14. Set EthReserveStabilizer exchange rate to $0.9925
15. Add DAI PSM to PCV Guardian

### CODE REVIEW

File: proposals/description/fip_56.ts [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    {
+      target: 'core',
+      values: '0',
+      method: 'createRole(bytes32,bytes32)',
+      arguments: [
+        '0x1749ca1ca3564d20da6efea465c2a5ae869a9e4b006da7035e688beb14d704e0',
+        '0x899bd46557473cb80307a9dabc297131ced39608330a2d29b2d52b660c03923e'
+      ],
+      description: 'Create PSM_ADMIN_ROLE'
+    },
```
Comment (eswak) [inline proposals/description/fip_56.ts:60]: This is not granted to anyone ?

**Review by eswak — DISMISSED** [review]

File: proposals/description/fip_56.ts (hunk 1 above)
Comment (ElliotFriedman) [reply proposals/description/fip_56.ts:60]: No, it is not granted to anyone but it is created and will be used later.

File: contract-addresses/permissions.json [hunk 2]
```diff
@@ -28,8 +25,7 @@
     "aaveEthPCVDripController",
     "compoundEthPCVDripController",
     "pcvGuardian",
-    "daiPCVDripController",
```
Comment (ElliotFriedman) [inline contract-addresses/permissions.json:31]: If the `daiPCVDripController` cannot pull DAI from the CompoundDAIPCVDeposit, then this whole system will not work.

We need to grant the PCVController role to the daiPCVDripController.

**Review by ElliotFriedman — CHANGES_REQUESTED** [review]

File: proposals/dao/fip_56.ts [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+  const tribeMinter: TribeMinter = contracts.tribeMinter as TribeMinter;
+  const tribeReserveStabilizer: TribeReserveStabilizer = contracts.tribeReserveStabilizer as TribeReserveStabilizer;
+
+  expect(await core.hasRole(role, tribeReserveStabilizer.address)).to.be.true;
+  expect(await tribe.minter()).to.be.equal(tribeMinter.address);
+  expect(await tribeMinter.isContractAdmin(tribeReserveStabilizer.address)).to.be.true;
+
+  const { daiPCVDripController, daiPSM, dai, compoundDaiPCVDeposit } = contracts;
+
+  expect(await daiPSM.underlyingToken()).to.be.equal(dai.address);
```
Comment (ElliotFriedman) [inline proposals/dao/fip_56.ts:190]: We should add a check to ensure that the daiPCVDripController is a PCVController in core.

Comment (Joeysantoro) [reply proposals/dao/fip_56.ts:190]: This is checked permissions.json

**Review by ElliotFriedman — APPROVED** [review]
One thing that stood out to me was the buffercap param being moved up from 10m to 20m, but that just makes the DAI psm more symmetrical in its buffers on DAI and FEI.

Other than that, LGTM!

**Pull request merged**